src/main/java/com/serenitydojo/Dog.java

```diff
@@ -48,4 +48,10 @@ public boolean isFed() {
     public void feed() {
         this.isFed = true;
     }
+
+
+    @Override
+    public String play() {
+        return "plays with bone";
```

Comment:
i am confused why we set a favorite toy and not use it. would be better like 
```suggestion
        return "plays with " + favoriteToy;
```
right? or is that like that in the exercise?

Reply:
i guess it doesn't matter because the exercise is about having the common Pet superclass and implementing abstract methods. nevermind :)

Reply:
Yeah, exactly